dt-users/users-endpoints.php

```diff
@@ -168,8 +168,8 @@ public function switch_preference( WP_REST_Request $request ) {
 
     public function app_switch( WP_REST_Request $request ) {
         $params = $request->get_params();
-        $user_id = get_current_user_id();
-        if ( isset( $params['app_key'] ) && ! empty( $params['app_key'] ) && $user_id ) {
+        $user_id = !empty( $params['user_id'] ) ? $params['user_id'] : get_current_user_id();
```

Comment:
@kodinkat what is keep a user from enabling or disabling other users apps?
(implement permissions)

Reply:
@kodinkat multipliers can update their own preferences, but should not be able to update someone else's.
Here the multiplier is blocked from this update.

Reply:
@kodinkat sorry, i wasn't clear here. We need to think through all the users here, not just the multipliers.

Simply allow the $user_id to be set manually here if the user can `manage_dt`, otherwise return an error if the $user_id is not the current logged in user